dll/win32/syssetup/syssetup.rc

```diff
@@ -108,6 +108,9 @@ IDR_GPL RT_TEXT "COPYING"
 #ifdef LANGUAGE_PT_BR
     #include "lang/pt-BR.rc"
 #endif
+#ifdef LANGUAGE_PT_PT
+    #include "lang/pt-PT.rc
```

Comment:
```suggestion
    #include "lang/pt-PT.rc"
```
The last double quotes are missing which is the cause why TravisCI fails to compile the build.

Reply:
Thanks for the suggestion Fraizeraust. I have already amended the error.